library/src-bootstrapped/scala/quoted/package.scala

```diff
@@ -2,9 +2,14 @@ package scala
 
 package object quoted {
 
-  object autolift {
-    implicit def autoToExpr[T: Liftable](x: T): Expr[T] = x.toExpr
-  }
+  /** An implicit conversion that removes the clutter of using `toExpr`
+  *   ubiquitously.
+  *
+  *   Note: this is just merely a convenience and PCP still holds. If the user
+  *   needs to maintain the visual cue of exprs between splices, disabling this
+  *   with `import scala.quoted.{autolift => _, _}`, enforces the uses of `toExpr`.
+  */
+  implicit def autolift[T: Liftable](x: T): Expr[T] = x.toExpr
```

Comment:
IMO having this on by default makes it an order of magnitude harder to get started using staging. I think we should wait for users to reach the "trust me, I know what I'm doing" state before throwing implicit conversions everywhere in their code...

Reply:
Exactly my thought. Staging is already confusing enough for newcomers, but if we blur the clean distinction between stages it just starts looking like dark magic. New users of LMS will attest.